tests/auth_tests/test_management.py

```diff
@@ -976,9 +976,9 @@ def test_create_permissions_checks_contenttypes_created(self):
         `post_migrate` handler ordering isn't guaranteed. Simulate a case
         where create_permissions() is called before create_contenttypes().
         """
-        # Warm the manager cache.
```

Comment:
Leave the `.`
(Makes the diff smaller if nothing else. But it's probably OK for it to be a fully-formed sentence. 🙂)

Reply:
Oh it's an oversight, good catch.